src/worker/vm/lazy.ts

```diff
@@ -14,38 +15,72 @@ import { clearError, processError } from '../../utils/db/error'
 import { disablePlugin, setPluginCapabilities } from '../../utils/db/sql'
 import { status } from '../../utils/status'
 import { createPluginConfigVM } from './vm'
-
 export class LazyPluginVM {
     initialize?: (hub: Hub, pluginConfig: PluginConfig, indexJs: string, logInfo: string) => Promise<void>
     failInitialization?: () => void
-    resolveInternalVm: Promise<PluginConfigVMResponse | null>
+    resolveInternalVm!: Promise<PluginConfigVMResponse | null>
+    totalAttemptsToInitialize: number
 
     constructor() {
+        this.totalAttemptsToInitialize = 0
+        this.initVM()
+    }
+
+    private initVM() {
```

Comment:
Nit: let's stick to `Vm` instead of `VM`

Reply:
Ah yes agreed